tags/faq/moddev.ytag

```diff
@@ -0,0 +1,8 @@
+type: text
+
+---
+
+You should use one of <#507982478276034570>, <#721100785936760876> or <#760651800570757150> for Fabric mod development questions.
```

Comment:
```suggestion
You should use one of <#507982478276034570>, <#721100785936760876> and <#760651800570757150> for Fabric mod development questions.
```

Reply:
Yes, but this is now outdated due to channel changes.